deploy_rln_contract.sh

```diff
@@ -6,13 +6,25 @@ set -e
 curl -L https://foundry.paradigm.xyz | bash && . /root/.bashrc && foundryup && export PATH=$PATH:$HOME/.foundry/bin
```

Comment:
Sure, I can add an environment variable to pass to the script to determine whether we want to deploy rlnv1 or rlnv2.
The contracts are deployed to different addresses, so the nwaku nodes registering their memberships will then also need some kind of check

Reply:
I mean, once this PR is merged, IMHO there is no need to care anymore about rln v1. So I would suggest removing all references to rlnv1, since rlnv2 will be the new and only RLN.

Reply:
Commit https://github.com/waku-org/waku-simulator/pull/45/commits/b0704ddee9e88f3563564e0ad8503c731dce8a15 updates deploy_rln_contract to only be valid for RLN-V2 as implemented in the waku-rlnv2-contract repo.